packages/react-scripts/config/paths.js

```diff
@@ -87,11 +95,11 @@ module.exports = {
   ownNodeModules: resolveOwn('node_modules'), // This is empty on npm 3
 };
 
-const ownPackageJson = require('../package.json');
-const reactScriptsPath = resolveApp(`node_modules/${ownPackageJson.name}`);
-const reactScriptsLinked =
-  fs.existsSync(reactScriptsPath) &&
-  fs.lstatSync(reactScriptsPath).isSymbolicLink();
+// const ownPackageJson = require('../package.json');
```

Comment:
I think this can be uncommented now. Was useful when react-scripts was in our codebase but now we're CRA, these checks are legit.